<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1224</width>
    <height>953</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ABS SYSTEM by Puzio&amp;Synoś Enterpreise ®</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Ravie</family>
            <pointsize>24</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>SYMULACJA UKŁDU ABS</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="label_PredkoscSamochodu">
              <property name="font">
               <font>
                <family>Broadway</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Prędkośc samochodu [m/s]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_PredkoscSamochodu">
              <property name="minimum">
               <number>15</number>
              </property>
              <property name="maximum">
               <number>75</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBar_PredkoscSamochodu">
              <property name="minimum">
               <number>15</number>
              </property>
              <property name="maximum">
               <number>75</number>
              </property>
              <property name="value">
               <number>24</number>
              </property>
              <property name="textVisible">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_PredkoscSamochoduValue">
              <property name="font">
               <font>
                <family>Broadway</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="label_MasaNaKolo">
              <property name="font">
               <font>
                <family>Broadway</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Masa samochodu na jedno koło [kg]</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_MasaNaKolo">
              <property name="minimum">
               <number>250</number>
              </property>
              <property name="maximum">
               <number>450</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBar_MasaNaKolo">
              <property name="minimum">
               <number>250</number>
              </property>
              <property name="maximum">
               <number>450</number>
              </property>
              <property name="value">
               <number>249</number>
              </property>
              <property name="textVisible">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_MasaNaKoloValue">
              <property name="font">
               <font>
                <family>Broadway</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="label_SrednicaKola">
              <property name="font">
               <font>
                <family>Broadway</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Promień koła samochodu [cm]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_SrednicaKola">
              <property name="minimum">
               <number>30</number>
              </property>
              <property name="maximum">
               <number>35</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBar_SrednicaKola">
              <property name="minimum">
               <number>30</number>
              </property>
              <property name="maximum">
               <number>35</number>
              </property>
              <property name="value">
               <number>29</number>
              </property>
              <property name="textVisible">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_SrednicaKolaValue">
              <property name="font">
               <font>
                <family>Broadway</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_KONSOLA">
              <property name="palette">
               <palette>
                <active>
                 <colorrole role="Text">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>0</red>
                    <green>0</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                </active>
                <inactive>
                 <colorrole role="Text">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>0</red>
                    <green>0</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                </inactive>
                <disabled>
                 <colorrole role="Text">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>120</red>
                    <green>120</green>
                    <blue>120</blue>
                   </color>
                  </brush>
                 </colorrole>
                </disabled>
               </palette>
              </property>
              <property name="font">
               <font>
                <family>Ravie</family>
                <pointsize>16</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string> USTAW WARTOŚCI I WCIŚNIJ SYMULUJ</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_SYMULUJ">
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>100</height>
                   </size>
                  </property>
                  <property name="palette">
                   <palette>
                    <active>
                     <colorrole role="Text">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>0</red>
                        <green>0</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </active>
                    <inactive>
                     <colorrole role="Text">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>0</red>
                        <green>0</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </inactive>
                    <disabled>
                     <colorrole role="Text">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>120</red>
                        <green>120</green>
                        <blue>120</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </disabled>
                   </palette>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>SYMULUJ</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Minimum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_ABS">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>200</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>200</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QCustomPlot" name="customPlot" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCustomPlot" name="customPlot2" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCustomPlot" name="customPlot3" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCustomPlot" name="customPlot4" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1224</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionO_programie"/>
    <addaction name="actionJak_korzysta"/>
   </widget>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionO_programie">
   <property name="text">
    <string>O programie</string>
   </property>
  </action>
  <action name="actionJak_korzysta">
   <property name="text">
    <string>Jak korzystać</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
